Update feed sync data configuration section (#348)
anchore/enterprise-docs: content/docs/using/cli_usage/feeds/feed_configuration.md

      ...
      feed_sync: 14400
```
### Feed Data Settings

Feed data configuration is set in the config.yaml file used by policy engine service. The `services.policy_engine.vulnerabilities.sync.data` section
of the configuration file controls the behavior of data to be synced. In addition, the data groups that can be synced depend on the `services.policy_engine.vulnerabilities.provider`, 

> further explained below.      

Reviewer: ```suggestion
and are explained in detail in the following sections.
```

> Note that the location and format of this config data changed slightly in Anchore Engine 0.10 to reflect some internal refactoring.  Ensure this config is synchronized between

Reviewer: ```suggestion
Note: The location and format of the config data changed slightly in Anchore Engine 0.10 to reflect some internal refactoring.  Ensure this 
```

containers if you are running more than one policy engine. This is usually handled for you by Helm Charts on Kubernetes, for example.

#### Grype  

By default, Anchore Enterprise is configured with `grype` as the `services.policy_engine.vulnerabilities.provider` and `grypedb` feed group enabled. 

> The `grypedb` feed group syncs a single grype database to the policy engine. 

Reviewer: ```suggestion
The `grypedb` feed group syncs a single Grype database to the policy engine. 
```

> A grype database contains data spanning multiple groups. Due to this encapsulation, it is not possible to enable or disable individual feed groups

Reviewer: ```suggestion
A Grype database contains data that spans multiple groups. Due to this encapsulation, it is not possible to enable or disable individual feed groups
```

> Anchore Enterprise will default to downloading the feed group from a publicly accessible url maintained by Grype https://toolbox-data.anchore.io/grype/databases/listing.json. 

Reviewer: ```suggestion
Anchore Enterprise will default to downloading the feed group from a publicly accessible URL maintained by Grype https://toolbox-data.anchore.io/grype/databases/listing.json. 
```

> The grype database available from this endpoint does not include third party/proprietary groups such as VulnDB and MSRC. 

Reviewer: ```suggestion
The Grype database available from this endpoint does not include third-party/proprietary groups such as VulnDB and MSRC. 
```

To get those groups, set `url` (or override the environment variable `ANCHORE_GRYPE_DB_URL`) to your local feed service.
```
services:
  ...
  policy_engine:
    ...
    vulnerabilities:
      provider: grype
      ...
      sync:
        ...
        data:
          grypedb:
            enabled: true
            url: ${ANCHORE_GRYPE_DB_URL}
```

#### Legacy

If `services.policy_engine.vulnerabilities.provider` is set to `legacy`, Anchore Enterprise will try to sync the following groups 
- `vulnerabilities` (enabled by default)
- `nvdv2` (enabled by default)
- `github` (enabled by default)
- `vulndb `
- `msrc`
- `packages`

Anchore Enterprise will default to downloading feed groups from Anchore's feed service hosted at https://ancho.re/v1/service/feeds and running in AWS in the

> us-west-2 region. This does not include third party/proprietary groups such as VulnDB and MSRC. 

Reviewer: ```suggestion
us-west-2 region. This does not include third-party/proprietary groups such as VulnDB and MSRC. 
```
